prediction_market_agent/agents/prophet_agent/deploy.py

```diff
@@ -137,9 +138,9 @@ class DeployablePredictionProphetGPT4TurboFinalAgent(DeployableTraderAgentER):
     def get_betting_strategy(self, market: AgentMarket) -> BettingStrategy:
         return KellyBettingStrategy(
             max_bet_amount=get_maximum_possible_bet_amount(
-                min_=1, max_=5, trading_balance=market.get_trade_balance(APIKeys())
```

Comment:
Previous strategy now:

```
| KellyBettingStrategy(max_bet_amount=5, max_price_impact=None)  |  566.814       |     3.25378     |   0.574048 |    0.267575 |              50 |       53.2538 |
```

(not in loss, but one of the worst strategies from profitable ones)